draft-thomson-rswg-syntax-change.md

```diff
@@ -133,9 +133,45 @@ This document has no IANA actions.
 
 --- back
 
+# Advice on Regenerating Publication Formats
+
+This document does not include specific guidance regarding the generation of
+publication formats from RFC XML source.  Decisions about how to maintain
+publication formats is not a matter governed by the policy in RFC 9280
+{{?RFC9280}}.  This section contains advice and considerations for the process
+of regeneration that came out of discussions of the policy changes in this
+document.
+
+Changes to the RFC XML for existing documents might result in changes to
+publication formats.  At the same time, the tools used to generate publication
```

Comment:
My argument here is that "HTML" is a publication format. That's not really metonymy for the published version

Reply:
I tried to capture that (rather subtle) distinction here, but probably failed.  Your point about generating, but not necessarily using, the HTML and whatnot was well made, I thought.